packages/stitch/src/stitchSchemas.ts

```diff
@@ -192,6 +195,8 @@ export function stitchSchemas<TContext = Record<string, any>>({
   return schema;
 }
 
+const subschemaConfigTransformerPresets = [isolateComputedFieldsTransformer, splitMergedTypeEntryPointsTransformer];
```

Comment:
The difference between the default and presets seems to be that defaults can be overridden, modified, perhaps by having their, but presets can't.

Also seems to be the case that the computed transformer operates on directives (version of computed directive when it was the only directive, before the stitching directives package), while the preset transformers operate on the merged type config itself.

When we push out a directive version of multiple entryPoints, I assume we will need to include the directive part in the default transformers rather than in the presets, parallel to computed, for those who wish to disable the automatic directive-based transformation or to customize it with a different directive name, etc.

Is that what you envision?

Reply:
Also, in a future major release, I imagine the computed directive transformer in default will be replaced by the stitching directives transformer?

Reply:
So, strictly speaking, the only built-in transformers should be these two presets:
- `isolateComputedFieldsTransformer`
- `splitMergedTypeEntryPointsTransformer`

We're basically looking at those as static fixtures within the stitching setup pipeline that will operate upon and configuration added by the user. The fact that those are structured as config transformers is simply a matter of stitching eating its own dogfood.

The ugly duckling here is the `computedDirectiveTransformer`, which is really just provided as backwards compatibility for the pre-SDL API `@computed` directive. The goal there would be to remove that transformer in the next major version, and to simply require the SDL package for all directive-based configuration.

Then to the question of how `entryPoints` would ultimately be translated into SDL directives: that would simply be an update to the `stitching-directives` package. The `stitchingDirectivesTransformer` just gets an update to write multi-entry SDL using `entryPoints` config. So, the SDL update can happen completely separately from this release, and the resulting `entryPoints` config it writes will flow directly into the code in this PR. I don't see a need to block this feature from rolling out until the supporting SDL transformer is written.

Reply:
In terms of writing multiple entry points, I think the different merge directives will have to be linked with the different key/computed directives. I was thinking we would need a directive only option “alias” where you could “name” the different entry points and reference that in the other directives. Any other ideas? Might make sense to start another issue or discussion as needed...